tests/test_evotuning.py

```diff
@@ -69,6 +68,13 @@ def test_length_batch_input_outputs():
     assert len(ys) == len(set([len(x) for x in sequences]))
 
 
+def test_get_batch_len():
+    batched_seqs = [["ABC", "ACD"], ["AABC", "EKQJ"], ["QWLRJK", "QJEFLK"]]
+    mean_batch_length, batch_lengths = get_batch_len(batched_seqs)
```

Comment:
Aghhhh, I should have caught this. Import needed, as the test fails without it. @ivanjayapurna, can you add in the import, do a `make format`, and push?